gravitee-node-api/src/main/java/io/gravitee/node/api/certificate/TrustStoreManager.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.node.api.certificate;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public interface TrustStoreManager {}
```

Comment:
Don't see any use of this interface? Even in VertxHttpServer, it returns null. Could you check on your side?

Reply:
Indeed, I've introduced these interfaces (`TrustStoreManager` and `KeyStoreManager`) just to make it clear that they will be stuck to each server in the future. I should probably remove them and introduce them later when the feature will be really developed 👍